static/skin/languages.json

```diff
@@ -0,0 +1,65 @@
+[
```

Comment:
`language.json` should not be versionned as it is generated.

Reply:
It is generated by the `static/generate_i18n_resources_list.py` script which is run manually.

Reply:
Why not generate it automatically ? As it is done for all other resources/templates.

Reply:
Which resource/template is generated automatically? If you mean preprocessing of resources with respect to cacheids or compiling them into C++ code (generating the `libkiwix-resources.{h,cpp}` files) - that's a different story. BTW, generating `languages.js` automatically will interfere with the said build steps which will have to be enhanced to take care of generated resources too.